__tests__/app.test.js

```diff
@@ -72,4 +72,45 @@ describe('GET /api/articles/:article_id', () => {
             expect(msg).toBe('ID Not Found');
         });
     });
+});
+
+describe('GET /api/articles/:article_id/comments', () => {
+    it('200: return the comments of the article with the specified ID ordered by most recent.', () => {
+        return request(app)
+        .get('/api/articles/3/comments')
+        .expect(200)
+        .then(({body}) => {
+            const {comments} = body;
+            expect(comments).toHaveLength(2);
+            expect(comments).toBeSortedBy('created_at');
+            comments.forEach(comment => {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: 3
+                });
+            });
+        });
+    });
```

Comment:
We could have the case where the id is valid but it just has no comments but wouldn't be an error, I would look to test this.